freqtrade/optimize/hyperopt_loss_sharpe_daily.py

```diff
@@ -0,0 +1,56 @@
+"""
+SharpeHyperOptLossDaily
+
+This module defines the alternative HyperOptLoss class which can be used for
+Hyperoptimization.
+"""
+from datetime import datetime
+
+from pandas import DataFrame
+import numpy as np
+
+from freqtrade.optimize.hyperopt import IHyperOptLoss
+
+
+class SharpeHyperOptLossDaily(IHyperOptLoss):
+    """
+    Defines the loss function for hyperopt.
+
+    This implementation uses the Sharpe Ratio calculation.
+    """
+
+    @staticmethod
+    def hyperopt_loss_function(
+        results: DataFrame,
+        trade_count: int,
+        min_date: datetime,
+        max_date: datetime,
+        *args,
+        **kwargs
+    ) -> float:
+        """
+        Objective function, returns smaller number for more optimal results.
+
+        Uses Sharpe Ratio calculation.
+        """
+        # get profit_percent and apply slippage of 0.1% per trade
+        results.loc[:, 'profit_percent'] = results['profit_percent'] - 0.0005
+
+        sum_daily = (
+            results.resample("D", on="close_time").agg(
+                {"profit_percent": sum}
+            )
+            * 100.0
+        )
+
+        if (np.std(sum_daily.profit_percent) != 0.):
+            sharp_ratio = (
+                sum_daily["profit_percent"].mean()
+                / sum_daily["profit_percent"].std()
+                * np.sqrt(365)
```

Comment:
i would decide on either `np.std()` - or `dataframe[].std()`. 
Using both seems within the same 5 lines is very odd ... 

Also best use either the pandas . notation (`dataframe.profit_percent`, or bracket notation `dataframe['profit_percent']`. 
Personally, i favor bracket notation, as that won't break if the column is accidentally called like a pandas function.

Reply:
Done, thanks.